components/dashboard/package.json

```diff
@@ -94,7 +96,7 @@
     "tailwind-underline-utils": "^1.1.2",
     "tailwindcss": "^3.4.3",
     "tailwindcss-filters": "^3.0.0",
-    "typescript": "~4.4.2",
+    "typescript": "^5.4.5",
```

Comment:
It seems it still builds just fine and am not aware of anything that this could break without it being reflected during build.

We need this to use the `Intl.ListFormat` API, which has ~95% browser support according to caniuse [[1](https://caniuse.com/mdn-javascript_builtins_intl_listformat)].

Reply:
Maybe we should continue with the migration in other components, but probably in a later PR.